gnosis/gp-ui
# [Explorer] Support for Bytes32 Token name/symbol methods

# Summary

Closes #728 

Added bytes32 symbol and name methods using the Bytes32 contract interface.

![image](https://user-images.githubusercontent.com/11525018/138787052-b5561429-f24f-4eda-880e-05151fca1931.png)

# To Test

1. Go to `orders/<order id>` page (i.e: `0xa2e6c6fe3a83080e0c911e3e75ef4fce8bddb2929f25685c9631255f88dcb6a704a66cbba0485d7b21af836f52b711401300fddb615f8e69`)
* You'll see token name and symbol for tokens on `Amount` row.
